core/CMakeLists.txt

```diff
@@ -624,7 +626,7 @@ if (UNIX)
         POST_BUILD
         COMMAND ${CMAKE_COMMAND}
         # to work around i#84 be sure to put a space after -D for 1st arg at least
-        ARGS -D lib=${lib_to_check}
+        ARGS -D lib_fileloc=${locvar_name}
```

Comment:
See below: I thought we'd pass in the ${locavar_name}.cmake path so that CMake_readelf.cmake can include it

Reply:
Done, see above.